### PR TITLE
Added oak-d-cm4 eeprom files

### PR DESCRIPTION
**Pull request by njezersek**

### CODE REVIEW

**Review by themarpe — APPROVED** [review]

File: batch/eeprom/DM2097_R1M1E1_oak_d_cm4_poe.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "batchName": "",
+    "batchTime": 0,
+    "boardConf": "nIR-C00M00-00",
+    "boardName": "DM2097",
+    "boardRev": "R1M1E1",
```
Comment (Luxonis-David) [inline batch/eeprom/DM2097_R1M1E1_oak_d_cm4_poe.json:6]: Change revision to: R3M1E4

File: boards/OAK-D-CM4-POE-C11.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+    "board_config":
+    {
+        "name": "OAK-D-CM4-C11",
```
Comment (Luxonis-David) [inline boards/OAK-D-CM4-POE-C11.json:4]: change to OAK-D-CM4-PoE-C11

File: batch/oak_d_cm4.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	"test_type": "OAK-D-CM4",
+	"variants": [
+		{
+			"title": "OAK-D CM4 (DM1097 R2M0E2)",
+			"description": "OAK-D CM4, based on DM1097 R2M0E2 board",
+			"eeprom": "eeprom/DM1097_R2M0E2_oak_d_cm4.json",
+			"board_config_file": "OAK-D-CM4.json"
+		},
+		{
+			"title": "OAK-D CM4 PoE (DM2097 R1M1E1)",
```
Comment (Luxonis-David) [inline batch/oak_d_cm4.json:13]: HW revision R3M1E4

File: boards/OAK-D-CM4-POE-C11.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        "revision": "R2M0E2",
+        "cameras":{
+            "CAM_A": {
+                "name": "rgb",
+                "hfov": 80.0,
```
Comment (Luxonis-David) [inline boards/OAK-D-CM4-POE-C11.json:9]: OAK-D-CM4-POE-C11 (w/OV9782) HFOV is the same as stereo HFOV so I suppose 71.86°

**Review by Luxonis-David — CHANGES_REQUESTED** [review]
Added few comments requesting changes

**Pull request merged**